### PR TITLE
PR: Add databags and adapt templates for localization

### PR DESCRIPTION
**Pull request by goanpeca**
This PR touches a lot of files, but the changes are needed in order to be able to localize all the content of the website.

### CODE REVIEW

File: templates/projects.html [hunk 1]
```diff
@@ -1,17 +1,18 @@
 {% extends "page.html" %}
+
 {% block title %}{{ this.title }}{% endblock %}
+
 {% block body %}
 <div class="row">
   <div class="col-sm-12 col-md-offset-1 col-md-10 col-lg-offset-2 col-lg-8">
     {{ this.body }}
-    <p>Each project of the BeeWare suite fills one of the following roles:</p>
```
Comment (freakboy3742) [inline templates/projects.html]: Has this line been lost?

File: models/home.ini [hunk 2]
```diff
@@ -25,6 +25,16 @@ type = flow
 label = Body blocks
 type = flow
 
+[fields.gutter_top]
+label = Gutter top
+description = Markwdown
```
Comment (freakboy3742) [inline models/home.ini]: Typo here (and in gutter_bottom).

Or should this be reStructuredText?

File: templates/blog.html [hunk 3]
```diff
@@ -22,9 +23,7 @@
 {% endblock %}
 {% block gutter %}
 <div class="col-sm-12 col-md-4 gutter">
-  <h3><p><a href="/news/buzz/atom.xml"><i class="fa fa-rss-square" aria-hidden="true"></i> RSS Feed</a></p></h3>
```
Comment (freakboy3742) [inline templates/blog.html]: Have we lost this RSS feed link?

File: templates/news.html [hunk 4]
```diff
@@ -19,25 +25,23 @@
...
       {% endif %}
-      on {{ child.pub_date }}
+      {{ t_on }} {{ child.pub_date }}
     </p>
     {% endfor %}
-    <a href="/news/buzz/">... more articles</a>
+    <a href="{{ '/news/buzz/'|url(alt=this.alt) }}">{{ t_more_articles }}</a>
   </div>
   <div class="col-md-12 col-lg-4 gutter">
-    <h3><p><a href="/news/buzz/atom.xml"><i class="fa fa-rss-square" aria-hidden="true"></i> RSS Feed</a></p></h3>
```
Comment (freakboy3742) [inline templates/news.html]: Has this atom link been lost?

File: templates/projects.html (hunk 1 above)
Comment (goanpeca) [inline templates/projects.html]: Nope, I added it to `{{ this.body }}` above

File: models/home.ini (hunk 2 above)
Comment (goanpeca) [inline models/home.ini]: I am hesitant of using rst on some places where we need to determine the level in an absolute way. On rst the level of a heading is relative to parsing the whole structure so:

```
Header level ?
--------------
```

On markdown on the other hand, its explicit. I will add a description to the model fields so we know what type to use.


```
## Header level 2
```

File: templates/blog.html (hunk 3 above)
Comment (goanpeca) [inline templates/blog.html]: No, it lives in the gutter top now
https://github.com/pybee/pybee.github.io/pull/164/files/b8a67f7c262350953a242d927515124f808c3af2#diff-827e86c354c59f395efb5c3f1f2e6ee5R26

File: templates/news.html (hunk 4 above)
Comment (goanpeca) [inline templates/news.html]: Same as before (moved to the gutter

File: templates/blog.html (hunk 3 above)
Comment (goanpeca) [inline templates/blog.html]: Nope, its here https://github.com/pybee/pybee.github.io/pull/164/files/b8a67f7c262350953a242d927515124f808c3af2#diff-827e86c354c59f395efb5c3f1f2e6ee5R26

**Comment by goanpeca** [comment]
Almost there @freakboy3742 only a PR needed to add a membership model (so it can also be localized in a saner manner), I will open that as a separate thing, cause this was getting too big. 

After that one I will add the Spanish translation one, cheers

**Comment by goanpeca** [comment]
@freakboy3742 I think this is ready

**Pull request merged**